### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by OmarShehata**
This fixes a bunch of typos, including a broken link to the Cesium logo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
-# cesium-concierge
+# Cesium Concierge
+
+__Hello! I'm Cesium Concierge, a GitHub bot for [Cesium](https://github.com/AnalyticalGraphicsInc/cesium). I automate
+common GitHub tasks such as welcoming first time contributors and reminding you to write tests (I'm great fun at parties).__
```
Comment (ggetz) [inline README.md]:
```suggestion
common GitHub tasks such as welcoming first time contributors and reminding you to write tests. (I'm great fun at parties!)__
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+
+You'll find repository-specific settings for each repository I watch under a `.concierge` directory. For example, here's CesiumJS's concierge settings:
```
Comment (ggetz) [inline README.md]:
```suggestion
You'll find repository-specific settings in each repository I watch under a `.concierge` directory. For example, here's CesiumJS's concierge settings:
```

**Comment by OmarShehata** [comment]
@ggetz I've applied your suggestions!

**Comment by ggetz** [comment]
Thanks @OmarShehata !

**Pull request merged**